website_snippet_marginless_gallery/views/snippets.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2015-2017 Tecnativa - Jairo Llopis <jairo.llopis@tecnativa.com>
+     Copyright 2019 Tecnativa - Cristina Martin R.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
+<odoo>
+    <template id="s_marginless_gallery" name="Marginless Gallery">
+        <section class="marginless-gallery pt32 pb32 text-center">
+            <div class="container">
+                <div class="row">
+                    <div class="col-lg-12 text-center">
+                        <h2 class="o_default_snippet_text pb16">
+                            <b>Our Gallery</b>
+                        </h2>
+                        <p class="lead o_default_snippet_text pb32">
+                            Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do.
+                        </p>
+                    </div>
+                    <section
+                        class="col-lg-3 col-12 pt32 pb32 oe_img_bg"
+                        style="background-image: url('/website/static/src/img/library/bridge.jpg');"
+                    >
+                        <div>
+                            <h4 class="o_default_snippet_text">Lorem ipsum</h4>
+                            <p class="o_default_snippet_text mb0">
+                                Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do.
+                            </p>
+                        </div>
+                    </section>
+                    <section class="col-lg-3 col-12 pt32 pb32 bg-alpha">
```

Comment:
```suggestion
                    <section class="col-lg-3 col-12 pt32 pb32 bg-primary">
```

Reply:
Class have been updated, thank you :)

Reply:
This is being handled in https://github.com/OCA/openupgradelib/pull/315 and https://github.com/OCA/OpenUpgrade/pull/3707